### DISCUSSION TITLE
msf exit code 0 with Docker-Compose

### DISCUSSION THREAD

**Discussion by obi-jon** [rapid7/metasploit-framework#15132] [Q&A] [answered]
Conditions:
docker-compose.override.yaml is renamed to docker-compose.override.yaml.disable
Host system is Ubuntu 20.04.2
user is in a group called docker with elevated privileges to bypass the need for sudo 
Using latest git clone as of today [1 May 2021]
local permissions for $HOME/.msf4 is 777

$ docker-compose up   #From inside the metasploit-framework folder
results in a "msf6 > metasploit-framework_ms_1 exited with code 0"
*see msfdocker_output.txt
[msfdocker_output.txt](https://github.com/rapid7/metasploit-framework/files/6410025/msfdocker_output.txt)

Question:
What am I missing?  Is there a configuration change I need to make?
I will note that the dev version exits the same way (docker-compose.override.yaml)

**Comment by smcintyre-r7** [comment, marked as answer]
It sounds like you probably want to open an issue for this rather than start a discussion post. The default issue template will help you get started by ensuring that you provide the necessary information to receive support.

**Reply by obi-jon** [reply]
Yeah, I just didnt want to open one until I could confirm it was actually an issue and not just an oversight of something I failed to read on my end.  Thanks.
